atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableClassRendererV2.java

```diff
@@ -110,6 +110,7 @@ public TableClassRendererV2(String packageName, Namespace namespace, String rawT
 
     public String render() {
         JavaFile javaFile = JavaFile.builder(packageName, this.buildTypeSpec())
+                .skipJavaLangImports(true)
```

Comment:
This matches what Conjure does:
https://github.com/palantir/conjure-java/search?q=skipJavaLangImports